### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by maso20160002**
hello

### CODE REVIEW

**Comment by zendynguyen2009** [comment]
i think...

**Comment by zendynguyen2009** [comment]
> hello

i so

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# Learn Git
\ No newline at end of file
+# Learn Git
+'''
+block
+'''
```
Comment (maso20160002) [inline README.md:4]: single comment

Comment (zendynguyen2009) [reply README.md:4]: hello

**Pull request merged**